public/config/README_configuration.md

```diff
@@ -29,81 +29,65 @@ The configuration file has four main sections and then several key-value pairs.
 The definition of the widgets placed on the page. The "widgets" property is
 an array of objects uniquely identified with the properties "type" and "id".
 
-#### widget CSS properties
+widget types:
+        socketValue: displayed float value
```

Comment:
Prefixing every type with "socket" doesn't seem to add any information value. It also implies these types only work with "socket", and no other mechanism. There are other widget types we'll implement, such as "Label", which doesn't have any robot data.

Reply:
removed prefix